buildenv/jenkins/README.md

```diff
@@ -51,15 +51,21 @@ This folder contains Jenkins pipeline scripts that are used in the OpenJ9 Jenkin
     - Windows on x86-64
         - Spec: x86-64_windows
         - shortname: win
+    - Windows on x86-64 largeheap/non-compressed references
+        - Spec: x86-64_windows_xl
+        - shortname: winlargeheap or winxl
```

Comment:
Personally I don't think we should have more than one shortname for a platform, that is just going to lead to confusion and an uglier regex in the jenkins job.

Reply:
I don't mind having only one. The original syntax for the shortname trigger was `<spec>largeheap` but I added `<spec>xl` as a shorter option. For backwards compatibilty I left the former there. There's no more regexes so the only added code is the bit in `SHORT_NAMES` in Pipeline-Build-Test-All.

Reply:
same story for the cmake shortnames

Reply:
I'm fine with the multiple names.